### PR TITLE
reorder project 

### PR DESCRIPTION
**Pull request by mjudeikis**
add dockerfile for docker strategy build

### CODE REVIEW

File: src/go/Dockerfile.azure-purge [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    && mv linux-amd64/glide /usr/bin/ \
+    && rm -r linux-amd64 \
+&& rm glide.tar.gz
+
+WORKDIR /go/src/github.com/azure-purge
+
+COPY . /go/src/github.com/azure-purge
+RUN ls -la /go/src/github.com/azure-purge
+RUN cd /go/src/github.com/azure-purge && \
+    make
```
[the remark this answers is not in the document]
Comment (mjudeikis) [reply src/go/Dockerfile.azure-purge]: did it openshift "stlyle" in release repo (chained builds) as we dont support this.

File: src/go/Dockerfile.azure-purge [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FROM golang:1.10-alpine
```
[the remark this answers is not in the document]
Comment (mjudeikis) [reply src/go/Dockerfile.azure-purge]: we cant do this without moving vendoring to the same dir.

[the remark this answers is not in the document]
Comment (kargakis) [reply src/go/Dockerfile.azure-purge]: ack

**Comment by mjudeikis** [comment]
cc @openshift/sig-azure

**Pull request merged**